lib/portlayer/metrics/container.go

```diff
@@ -0,0 +1,327 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
```

Comment:
Agree to an extent..also from my perspective the vmomiGW would be ideal place for this type of infra interaction.